ZenPacks/zenoss/Import4/migration.py

```diff
@@ -287,16 +287,17 @@ def reportHeartbeat(self):
 
     # socket must be supplied because
     # we use two different sockets for zep and zodb
-    def restoreMySqlDb(self, sql_path, db, socket, status_key='sql'):
+    def restoreMySqlDb(self, sql_path, db, socket, status_key='sql', status_max=0):
         """
         Create MySQL database if it doesn't exist.
         """
 
         # obtain the number of insert statements again
-        status_max = int(subprocess.check_output(
-            'egrep "^INSERT INTO" %s | wc -l' % sql_path, shell=True))
         if status_max <= 0:
-            log.error("Cannot find any INSERT statement in %s" % sql_path)
+            status_max = int(subprocess.check_output(
```

Comment:
Yes.. to make sure that the sql has some inserts.

On Mon, Aug 17, 2015 at 12:03 PM, Andrew Lapidas <notifications@github.com>
wrote:

> In ZenPacks/zenoss/Import4/migration.py
> <https://github.com/zenoss/ZenPacks.zenoss.Import4/pull/83#discussion_r37210315>
> :
>
> >          if status_max <= 0:
> > -            log.error("Cannot find any INSERT statement in %s" % sql_path)
> > +            status_max = int(subprocess.check_output(
>
> What't the point of this? To make sure there are at least some INSERT
> statements in the .sql file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zenoss/ZenPacks.zenoss.Import4/pull/83/files#r37210315>
> .
>



-- 
Thanks,
----
Alex